tarot_juicer/settings.py

```diff
@@ -33,20 +33,25 @@
 
 # SECURITY WARNING: don't run with debug turned on in production!
 
-if os.environ.get('DJANGO_DEBUG', '') == 'True':
+if os.environ.get('DJANGO_DEBUG', '') != 'False':
     # These are testing settings:
     DEBUG = True 
-    SECURE_HSTS_SECONDS = 10
+    SECURE_HSTS_SECONDS = 0
     SECURE_SSL_REDIRECT = False
     SESSION_COOKIE_SECURE = False
     CSRF_COOKIE_SECURE = False
+    SECURE_HSTS_PRELOAD = False
+    SECURE_HSTS_INCLUDE_SUBDOMAINS = False
+    print("\nMode: ", '\033[91m' + 'Not Secure' + '\033[0m', "\n") # Added colored output as red
 else:
     # These are prod settings:
-    DEBUG = True # Set to `False` for prod when done testing prod (for when the project is finally Live)
-    SECURE_HSTS_SECONDS = 10
-    SECURE_SSL_REDIRECT = True
-    SESSION_COOKIE_SECURE = True
-    CSRF_COOKIE_SECURE = True
+    DEBUG = False # Set to `False` for prod when done testing prod (for when the project is finally Live)
+    # SECURE_HSTS_SECONDS = 3600
+    # SECURE_SSL_REDIRECT = True
+    # SESSION_COOKIE_SECURE = True
+    # CSRF_COOKIE_SECURE = True
+    # SECURE_HSTS_PRELOAD = True
```

Comment:
@UmarGit I'm very curious to understand why these lines were commented out because without these security settings in place, Django won't work in the cloud. Anyways, I merged the changes into master and then pushed up to Heroku. As expected, it is broken: https://tarot-prod.herokuapp.com/ 

It's showing error 500. It's not any more specific than that because `DEBUG` is now set to False, so that makes sense to me. But why did you decide to comment out all these settings for prod?

Reply:
:rofl: Oh Sorry, I forgot to un-comment that